pkg/cmd/server/origin/master.go

```diff
@@ -307,9 +303,6 @@ func (c *MasterConfig) withConsoleRedirection(handler, assetServerHandler http.H
 	if assetConfig == nil {
 		return handler
 	}
-	if !c.WebConsoleEnabled() || c.WebConsoleStandalone() {
```

Comment:
was it opt-in and is not anymore?

Reply:
> was it opt-in and is not anymore?

Yeah.  It's moving in 3.9.  I don't think there's any plan to continue supporting the old path.  They've even stopped pushing bindata updates.